#
# Submitted-By: Erwin Rol, 2006-0326
# Committed-By: Erwin Rol
#
# Error:
# mini shell script uses #!/bin/bash which is aproblem for busybox
#
# Description:
# replace /bin/bash with /bin/sh
#
# State:
# Fixed in upstream SVN
#
--- initng-0.6.0/tools/initng-segfault.orig	2006-03-26 18:06:46.000000000 +0200
+++ initng-0.6.0/tools/initng-segfault	2006-03-26 18:06:54.000000000 +0200
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 echo "Initng seems to have segfaulted."
 echo
